Block/Checkout/Overview.php

```diff
@@ -117,8 +118,9 @@ protected function _toHtml()
     {
 
         $aboveThreshold = $this->quoteHelper->aboveThreshold($this->quoteHelper->getSessionQuote()->getGrandTotal());
+        $disabledSkus = !$this->quoteHelper->checkDisabledForSku('',true);
 
-        if ($this->quoteHelper->isEnabledOnCOP() && $aboveThreshold) {
+        if ($this->quoteHelper->isEnabledOnCOP() && $aboveThreshold && $disabledSkus) {
```

Comment:
the `disabledSkus` name seems like an array, so I think it should be changed to be a boolean type name here and elsewhere

Reply:
what would be a boolean type name?

Reply:
like isDisabledBySku or something like that